inc/sitemaps/interface-sitemap-cache-data.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @package WPSEO\XML_Sitemaps
+ */
+
+/**
+ * Cache Data interface
+ */
+interface WPSEO_Sitemap_Cache_Data_Interface {
+
+	/** Status for normal, usable sitemap. */
+	const OK = 'ok';
+
+	/** Status for unusable sitemap. */
+	const ERROR = 'error';
+
+	/** Status for unusable sitemap because it cannot be identified. */
```

Comment:
The documentation for `unknown` and `error` are very similar, except for a bit more information on `unknown`. Could you please elaborate why in this case it is unknown and not an error?